components/Shared/HeaderAgent.vue

```diff
@@ -66,45 +62,49 @@ export default {
     text-decoration: none;
 }
 
-.agent-dropdown-menu h2 {
+.agent-dropdown .dropdown-menu h2 {
     color: #103A5B;
     font-size: 18px;
     font-weight: 700;
     text-align: left;
     margin-bottom: 15px;
 }
 
-.agent-dropdown-menu h3 {
+.agent-dropdown .dropdown-menu h3 {
     font-size: 14px;
     color: #103A5B;
     font-weight: 600;
 }
 
-.agent-dropdown-menu input {
+.agent-dropdown .dropdown-menu input {
     border-radius: 50px;
 }
 
-.agent-dropdown-menu input::placeholder {
+.agent-dropdown .dropdown-toggle::after {
+    display: none !important;
+}
+
+.agent-dropdown .dropdown-menu input::placeholder {
     color: #CBCBCB;
     padding-left: 10px;
 }
 
-.agent-dropdown-menu form {
+.agent-dropdown .dropdown-menu form {
     border-bottom: 1px solid #EFEFEF;
     margin-bottom: 25px;
     margin-top: 30px;
     padding-bottom: 25px;
 }
 
-.agent-dropdown-menu .agent-form-label {
+.agent-dropdown .dropdown-menu .agent-form-label {
     font-size: 13px;
 }
 
-.agent-dropdown-menu a {
+.agent-dropdown .dropdown-menu a {
     color: #1B75BB;
 }
 
-.agent-dropdown-menu p {
+.agent-dropdown .dropdown-menu p {
     line-height: 1.5em;
```

Comment:
should use rem instead of em for easier control

Reply:
updated.